roles/chronos/defaults/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+# right now there is an issue with chronos zookeeper connection with auth, so leave it blank for now
```

Comment:
If we run a chronos with `--zk-hosts chronos:8WDxe30zbCzcuZJc@zookeeper.service.consul:2181` param we get the following error:
```
May 29 09:52:09 clean-deploy-test-01 chronos[17850]: 2015-05-29 09:52:09,300:17850(0x7fd7f1374700):ZOO_INFO@zookeeper_init@786: Initiating client connection, host=chronos:8WDxe30zbCzcuZJc@z
ookeeper.service.consul:2181 sessionTimeout=10000 watcher=0x7fd7fae3c1e0 sessionId=0 sessionPasswd=<null> context=0x7fd818021d40 flags=0
May 29 09:52:09 clean-deploy-test-01 chronos[17850]: F0529 09:52:09.333401 18132 zookeeper.cpp:113] Failed to create ZooKeeper, zookeeper_init: No such file or directory [2]
```

`host=chronos:8WDxe30zbCzcuZJc@zookeeper.service.consul:2181` -  i think this is the cause of the issue.

The marathon log looks a little different:
```
May 29 10:12:27 clean-deploy-test-01 marathon[20835]: 2015-05-29 10:12:27,840:20835(0x7fb639368700):ZOO_INFO@zookeeper_init@786: Initiating client connection, host=zookeeper.service.consul:
2181 sessionTimeout=10000 watcher=0x7fb642e3c1e0 sessionId=0 sessionPasswd=<null> context=0x7fb648beeb00 flags=0
```

i cant say if marathon just drops the auth part and this makes it start, or its an error in the chronos args parser